app/models/trip.rb

```diff
@@ -28,4 +29,82 @@ def self.worst_weather
       .count(:condition_id)
       .min_by { |_k, v| v }[0]
   end
+
+  def self.average_duration
+    average(:duration)
+  end
+
+  def self.min_duration
+    minimum(:duration)
+  end
+
+  def self.max_duration
+    maximum(:duration)
+  end
+
+  def self.group_by_month
+    group("DATE_TRUNC('month', start_date)").count
+  end
+
+  def self.beautify_group_by_month
+    months = group_by_month
+
+    formatted = months.keys.map do |month|
```

Comment:
This seems ruby-ish and not AR... maybe there's a view helper that could achieve this?